pkg/cmd/start/start_test.go

```diff
@@ -270,8 +296,20 @@ func TestPipelineStart(t *testing.T) {
 			params[p.Name] = p.ValueTemplate
 		}
 
-		if tc.verify != nil {
-			tc.verify(o, params)
+		if tc.verifyParams != nil {
+			tc.verifyParams(o, params)
+		}
```

Comment:
Same cleanup applies here: If you remove the verifyParams from the "file" case (which doesn't do anything) you can move move `if tc.verifyParams != nil {` up to replace `if name != "file" && name != "add-custom-envs" {`

Reply:
https://github.com/jenkins-x-plugins/jx-pipeline/pull/510/commits/e7f5dbd6195b4dbb39d9bfe15102ac01db267213 should do the needful

Reply:
Not quite. I'll fix it.

Reply:
I'm not sure but did you mean [this](https://github.com/jenkins-x-plugins/jx-pipeline/pull/510/commits/a4224158d11dcf7465842417b5fce3ba473f14b4)?

Reply:
@msvticket I spent thinking on how can I make it better but the last commit is only what I could think of.
Let me know if I'm not on the right track